src/styles/BookList.module.css

```diff
@@ -5,34 +5,108 @@
 }
 
 .book {
-  border: 1px solid black;
+  border: 1px solid rgb(211, 207, 207);
   margin: 1rem;
   list-style-type: none;
   padding: 1rem;
-  border-radius: 5px;
+  height: 27vh;
+  border-radius: 4px;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
 }
 
 .btnContainer {
   margin-top: 10px;
 }
 
 button {
-  margin-right: 0.25rem;
+  margin-right: 2rem;
+  color: #0290ff;
+  border: none;
+  background: transparent;
 }
 
 form {
-  margin: 2rem 1rem;
+  padding: 0 4rem 4rem 4rem;
+  background-color: rgb(200, 200, 200, 0.1);
+}
+
+.form {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 1rem;
+}
```

Comment:
- [ ] Kindly ensure that the elements spacing, the color and the size of the input fields and button in your form matches the given design.
- [ ] Also, please add the horizontal line highlighted in red as it is a part of the design

### Your design
<img width="1369" alt="Screenshot 2023-06-15 at 22 20 05" src="https://github.com/forLoop94/book-store/assets/64774936/f0aaafc1-96d1-4ab8-984a-8d6233e75df1">

### Given design

<img width="1075" alt="Screenshot 2023-06-15 at 22 41 37" src="https://github.com/forLoop94/book-store/assets/64774936/8ea17fb0-aafb-4ae6-a96f-0c5634fdc82a">